src/sentry/models/debugfile.py

```diff
@@ -361,6 +362,23 @@ def _analyze_progard_filename(filename: str) -> Optional[str]:
         return None
 
 
+@region_silo_only_model
+class ProguardArtifactRelease(Model):  # type: ignore
+    __include_in_export__ = False
+
+    organization_id = BoundedBigIntegerField(db_index=True)
+    project_id = BoundedBigIntegerField(db_index=True)
+    release_name = models.CharField(max_length=250)
+    proguard_uuid = models.UUIDField()
+    date_added = models.DateTimeField(default=timezone.now)
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_proguardartifactrelease"
+
+        unique_together = (("organization_id", "project_id", "release_name"),)
```

Comment:
This uniqueness constraint implies that we can't have multiple proguard uuids with the same org, project and release, is this wanted?

Reply:
Thx for point that out, it's changed now to include "proguard_uuid"

`"unique_together": {("organization_id", "project_id", "release_name", "proguard_uuid")},`

It should be possible to have a proguard_uuid reference multiple releases, hopefully that makes sense